config/checkstyle/checkstyle.xml

```diff
@@ -193,7 +193,7 @@
     <module name="CustomImportOrder">
       <property name="sortImportsInGroupAlphabetically" value="true"/>
       <property name="separateLineBetweenGroups" value="true"/>
-      <property name="customImportOrderRules" value="THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE###STATIC"/>
+      <property name="customImportOrderRules" value="STATIC###THIRD_PARTY_PACKAGE"/>
```

Comment:
That is the HMCTS standard, most projects use it, what's the reason for the change?
https://github.com/search?q=org%3Ahmcts+customImportOrderRules&type=code

Reply:
I had such a fight with Checkstyle and IntelliJ. I changed it to what tribunals-api uses, if it's standard across I will change it back.